plugin/db/pg/sync.go

```diff
@@ -278,17 +268,51 @@ func (driver *Driver) getUserList(ctx context.Context) ([]db.User, error) {
 
 	for rows.Next() {
 		var role string
-		var attr string
+		var super, inherit, createRole, createDB, canLogin, replication, bypassRLS bool
+		var rolValidUtil sql.NullString
```

Comment:
Is leaving out rolconnlimit intentional?  https://github.com/bytebase/bytebase/pull/2374/files

Reply:
Yeah, this boolean isn't used for deriving the attribute in PostgreSQL. We could always add it later especially when it's ready to handle membership/ACL.